src/index/updater/inscription_updater.rs

```diff
@@ -116,10 +116,13 @@ impl<'a, 'db, 'tx> InscriptionUpdater<'a, 'db, 'tx> {
     if inscriptions.iter().all(|flotsam| flotsam.offset != 0)
       && Inscription::from_transaction(tx).is_some()
     {
+      let mut fee = input_value;
+      fee -= tx.output.iter().map(|txout| txout.value).sum::<u64>();
```

Comment:
Also this can be inlined:

```suggestion
      let fee = input_value - tx.output.iter().map(|txout| txout.value).sum::<u64>();
```